sinks/gcm/metrics.go

```diff
@@ -92,4 +91,76 @@ var allMetrics = []SupportedMetric{
 			return int64(stat.Memory.WorkingSet)
 		},
 	},
+	{
+		MetricDescriptor: MetricDescriptor{
+			Name:        "memory/page_faults",
+			Description: "Number of major page faults",
+			Type:        MetricGauge,
+			ValueType:   ValueInt64,
+		},
+		HasValue: func(spec *cadvisor.ContainerSpec) bool {
+			return spec.HasMemory
```

Comment:
Yes, I think that makes sense. Similar to has_<field> in proto. Will file an issue :)